solutions/gold/baltic-09-candy.mdx

```diff
@@ -0,0 +1,152 @@
+---
+id: baltic-09-candy
+source: Baltic OI 2009
+title: Candy Machine
+author: Panos Paskalis
+---
+
+[Official Analysis](http://www.csc.kth.se/contest/boi/candy-spoiler.pdf)
+
+## Explanation
+
+### Observation
+
+If there is a candy $i$ $(s_i, t_i)$ and a candy $j$ $(s_j, t_j)$, with
+$t_i < t_j$, then $j$ can be caught by the same wagon as $i$, if the distance
+between them can be covered by the wagon in $t_j - t_i$ time. That is:
+
+$$
+|s_j - s_i| \leq t_j - t_i \iff
+t_i - t_j \leq s_j - s_i \leq t_j - t_i \iff
+$$
+$$
+\begin{cases}
+    s_i + t_i \leq s_j + t_j \text{ and } \\
+    s_j - t_j \leq s_i - t_i \\
+\end{cases}
+$$
+Since $t_i \geq 0$,
+$$
+s_i - t_i \leq s_i + t_i
+$$
+In total:
+$$
+s_j - t_j \leq s_i - t_i \leq s_i + t_i \leq s_j + t_j
+$$
```

Comment:
these are just letters and symbols
they mean basically nothing lol
try to explain in english what you're doing throughout

Reply:
I tried to use more text now.